### ISSUE TITLE
implicit AtPath[Duration] can be AtPath[FiniteDuration]

### ISSUE THREAD

**Issue by tkawachi** [kxbmap/configs#6]
The implementation requires a long value to be read. So it can be FiniteDuration.

````
  implicit val durationAtPath: AtPath[Duration] = atPath {
    Duration fromNanos _.getDuration(_, TimeUnit.NANOSECONDS)
  }
````

**Comment by kxbmap** [comment]
Yes, you are right. I'll fix it in 0.3.x.

**Comment by kxbmap** [comment]
Fixed in https://github.com/kxbmap/configs/commit/c5dabbdf2560a84c382fd59b869ee9934a2989ac and merged into master.
